### ISSUE TITLE
Migrate e-mail sending to new Sendgrid account

### ISSUE THREAD

**Issue by jayzes** [denverstartupweek/dsw-site#138]
- [x] Change Sendgrid credentials on Heroku and deprovision add-ons
- [x] Import existing list from Emma to Sendgrid (including additional attributes)
- [x] Set up automatic subscriptions to Sendgrid in `ListSubScriptionJob` the same as Emma
- [ ] (optional) Migrate existing transactional e-mails to Sendgrid templates so others can edit the copy if they want.

**Comment by jayzes** [comment]
Waiting on leveling-up of the new Sendgrid account so I can migrate things over.

**Comment by jayzes** [comment]
Closing this one and moving the last one to a new issue.
